test/server-bare.test.js

```diff
@@ -209,21 +209,17 @@ describe('Session specific tests', () => {
   // We save and then load, which creates a new session.
```

Comment:
Can we add unit tests that covers the following scenario?
It'll ensure all this logic from silently breaking for customers.

1) Zigbee + Matter templates are pre-loaded.
    * verify Zigbee config loads correctly
    * verify Matter config loads correctly
    * if no matching packages are found, what should happen here? 
2) Only Zigbee template is pre-loaded
    * verify Matter config loads correctly
3) No templates are pre-loaded
    * verify Zigbee config loads correctly
    * verify Matter config loads correctly

Reply:
We should. I will log an issue.

Reply:
Created: https://github.com/project-chip/zap/issues/817